cmd/opm/alpha/veneer/semver.go

```diff
@@ -18,17 +18,24 @@ import (
 func newSemverCmd() *cobra.Command {
 	output := ""
 	cmd := &cobra.Command{
-		Use:   "semver <filename>",
-		Short: "Generate a file-based catalog from a single 'semver veneer' file",
-		Long:  `Generate a file-based catalog from a single 'semver veneer' file`,
-		Args:  cobra.MinimumNArgs(1),
+		Use:   "semver [FILE]",
+		Short: "Generate a file-based catalog from a single 'semver veneer' file \nWith no File, or when FILE is -, the file is read from standard input",
+		Long:  "Generate a file-based catalog from a single 'semver veneer' file \nWith no File, or when FILE is -, the file is read from standard input",
```

Comment:
Nit:  consistency of parameter
```suggestion
		Short: "Generate a file-based catalog from a single 'semver veneer' file \nWhen FILE is '-' or not provided, the veneer is read from standard input",
		Long:  "Generate a file-based catalog from a single 'semver veneer' file \nWhen FILE is '-' or not provided, the veneer is read from standard input",
```